### PR TITLE
[msquic] Initial integration

### PR DESCRIPTION
**Pull request by ami-GS**

### CODE REVIEW

File: projects/msquic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+ADD https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb packages-microsoft-prod.deb
```
Comment (jonathanmetzman) [inline projects/msquic/Dockerfile:19]: We're on Ubuntu 20, this is weird.

File: projects/msquic/project.yaml [hunk 2]
```diff
@@ -2,5 +2,11 @@ homepage: "https://github.com/microsoft/msquic"
 language: c
 primary_contact: "msquicinfra@microsoft.com"
 auto_ccs:
+  - "quicdev@microsoft.com"
```
Comment (jonathanmetzman) [inline projects/msquic/project.yaml:5]: note: This will get emails about bugs but members of this list wont be able to access them.

**Review by jonathanmetzman — APPROVED** [review]
LGTM. Lets hope coverage fail was a fluke

**Comment by jonathanmetzman** [comment]
It looks like linking coverage builds is running out of RAM. In production we have more RAM. I can merge this if you want or you can try to fix (maybe adjust parallelism?)

**Comment by ami-GS** [comment]
Thanks for reviewing.
As far as I checked, the parallelism should be 1 by default. it would be great if you can merge with the issue.

**Pull request merged**